### PR TITLE
Add news post for playtest-20180102.

### PR DESCRIPTION
**Pull request by pchote**
TODO:
- [x] Final changelog link
- [x] Battlefield news
- [x] Playtest release ID

20180101-fullpage

### CODE REVIEW

File: content/news/playtest-20180101.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+We make these playtests available to get feedback from the community about the future direction of OpenRA, and we value your opinions.
+Let us know what you think and please report any bugs you discover via [our forum](http://www.sleipnirstuff.com/forum/viewforum.php?f=80), our [GitHub issue tracker](https://github.com/OpenRA/OpenRA/issues), or in the comments below.
+
+OpenRA playtest builds do not overwrite the release version if you are on Windows or macOS, and you can switch between the two when joining an online server. So head over to our [download page](/download/) and give it a try!
+
+The last development cycle has brought some significant improvements for modders, but we recommend that projects currently based on the stable release to hold off upgrading for now. Experienced OpenRA modders well know the difficulties of updating your code for a new OpenRA release, and this playtest includes more changes than most. We are planning major improvements to our semi-automatic mod-upgrader to be included in a future playtest, and we will update the [OpenRA Mod SDK](https://github.com/OpenRA/OpenRAModSDK) when these changes are ready. Stay tuned for more info!
```
Comment (penev92) [inline content/news/playtest-20180101.md]: Need to fix the "recommend that someone to do something" construct.

File: content/news/playtest-20180101.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+We are excited to welcome the new year with a new [OpenRA playtest](/downloads/)!
+During the last 5 months we merged over 700 commits from 42 developers, with each commit working towards fixing bugs, improving code quality, or adding new features that improve the single- and multi-player experience.
```
Comment (reaperrr) [inline content/news/playtest-20180101.md]: Maybe 'contributors' instead of 'developers'?

File: content/news/playtest-20180101.md (hunk 1 above)
Comment (reaperrr) [inline content/news/playtest-20180101.md]: In that first sentence I'd remove the 'to' before and add 'from' after the 'hold off'.

File: content/news/playtest-20180101.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+As always, we suggest reading the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog) for more information and to view the full list of changes.
+We make these playtests available to get feedback from the community about the future direction of OpenRA, and we value your feedback.
+Let us know what you think and please report any bugs you discover via [our forum](http://www.sleipnirstuff.com/forum/viewforum.php?f=80), our [GitHub issue tracker](https://github.com/OpenRA/OpenRA/issues), or in the comments below.
+
+OpenRA playtest builds do not overwrite the release version if you are on Windows or macOS, and you can switch between the two when joining an online server. So head over to our [download page](/download/) and give it a try!
+
+This development cycle brought significant improvements for community mods, but we recommend that modders wait for a future playtest before updating your projects.  We plan to merge a new version of our semi-automated rule upgrader before the final release, which will siimplify the process of updating your [Mod SDK](https://github.com/OpenRA/OpenRAModSDK) based projects to the latest engine version. Stay tuned for more info!
```
Comment (abcdefg30) [inline content/news/playtest-20180101.md]: `siimplify`

File: content/news/playtest-20180101.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+<div style="text-align:center" markdown="1">
+![AI Airstrike](/images/news/20180101-airstrike.png)
+
+Improvements to the AI support power targeting mean that this will be a rarer sight.
+ </div>
+
+Other notable changes include:
+
+* New lobby options to disable bounty, build radius, and re-packing Construction Yards
+* Units will no longer target units revealed by the Red Alert GPS Satellite
```
Comment (abcdefg30) [inline content/news/playtest-20180101.md]: What about replacing the second `units` by `enemies` (so we don't have units twice in the same sentence)?

File: content/news/playtest-20180101.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+The new server player list is one of several improvements to the multiplayer UI
+</div>
+
+Of course, we realize that the vast majority of OpenRA players do not play online. We haven't forgotten about you either!
+Improvements for the singleplayer experience include:
+
+ * The full Harkonnen campaign in D2K
+ * Refinements and improvements to several campaign missions in Red Alert and Tiberian Dawn
+ * A fix for the AI cheating when it targets superweapons and support powers
```
Comment (abcdefg30) [inline content/news/playtest-20180101.md]: `targets` might be confusing here. At first I thought the AI is targeting the powers. We could use `fires` instead.

File: content/news/playtest-20180101.md (hunk 3 above)
Comment (abcdefg30) [inline content/news/playtest-20180101.md]: "that modders wait [...] before updating your projects." 3rd person versus 2nd person? your -> their?

File: content/news/playtest-20180101.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+ * The server browser shows the name/faction/team of connected players
+ * The server browser is now also available as a tab in the server lobby
+ * The main menu and server browser will notify you if a new release is available
+ * The Global Chat has been removed from the server browser and lobby
+ * A chat dialog has been added to the in-game menu
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20180101-serverplayers.png" width="600" alt="Server player list">
+
+The new server player list is one of several improvements to the multiplayer UI
```
Comment (abcdefg30) [inline content/news/playtest-20180101.md]: Missing full stop.

File: content/news/playtest-20180101.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
+We are excited to welcome the new year with a new [OpenRA playtest](/downloads/)!
+More than 700 commits from 42 contributors were merged over the last five months, with each commit helping to fix bugs, improve code quality, or add new features that improve the single- and multi-player experience.
+
+This playtest is a timely opportunity to get community participation, as we celebrate a massive record of games played. At the end of November we passed 500,000 completed multiplayer games since August 2014, when we started recording online server statistics. This is a huge accomplishment for our community, congratulations!
+
+The main focus of this development cycle was modernizing core parts of the engine and game code to support the work-in-progress Tiberian Sun mod and other community projects. These shouldn't have visibly changed our released mods, but big changes always have a risk of introducing unexpected bugs or side-effects. Please note that changes to the way hotkeys are managed means you will need to redefine any changes you made in the Input settings.
```
Comment (abcdefg30) [inline content/news/playtest-20180101.md]: `changes means` versus `changes mean`?

**Comment by pchote** [comment]
Updated and added battlefield news:

screen shot 2018-01-01 at 14 13 43

File: content/news/playtest-20180101.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+The Map Options tab includes several new settings, and can be customized by scripted maps.
+</div>
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20180101-winmacos.png" width="600" alt="Windows and macOS improvements">
+
+Platform integration fixes include "Pin to taskbar" support on Windows and 64 bit support for macOS.
+</div>
+
+As always, we suggest reading the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog) for more information and to view the full list of changes.
```
Comment (penev92) [inline content/news/playtest-20180101.md]: "reading and to view" sounds meh

File: content/news/playtest-20180101.md (hunk 3 above)
Comment (reaperrr) [reply content/news/playtest-20180101.md]: Also, first sentence: In my opinion 'their projects' would be more consistent with 'modders'.

**Comment by pchote** [comment]
Updated.

**Comment by penev92** [comment]
LGTM 👍

**Pull request merged**